test/ValidatorChainTest.php

```diff
@@ -103,45 +103,6 @@ public function testBreakChainOnFailure()
         $this->assertEquals(['error' => 'validation failed'], $this->validator->getMessages());
     }
 
-    /**
-     * Ensures that if we specify a validator class basename that doesn't
-     * exist in the namespace, is() throws an exception.
-     *
-     * Refactored to conform with ZF-2724.
-     *
-     * @group  ZF-2724
-     * @return void
-     */
-    public function testStaticFactoryClassNotFound()
-    {
-        $this->setExpectedException('Zend\ServiceManager\Exception\ServiceNotFoundException');
-        StaticValidator::execute('1234', 'UnknownValidator');
-    }
-
-    public function testIsValidWithParameters()
-    {
-        $this->assertTrue(StaticValidator::execute(5, 'Between', [1, 10]));
-        $this->assertTrue(StaticValidator::execute(5, 'Between', ['min' => 1, 'max' => 10]));
-    }
-
-    public function testSetGetMessageLengthLimitation()
-    {
-        AbstractValidator::setMessageLength(5);
```

Comment:
There is another test as this one in `StaticValidatorTest` which is incorrect too. This must me moved to BetweenTest